docs/src/lib/representations.md

```diff
@@ -142,7 +142,7 @@ dim(::EmptySet)
 ZeroSet
 dim(::ZeroSet)
 σ(::AbstractVector{Float64}, ::ZeroSet)
-∈(::AbstractVector{Float64}, ::ZeroSet)
+∈(::AbstractVector{Float64}, ::ZeroSet{Float64})
```

Comment:
let's merge this unrelated change via another branch? one question: don't you need this annotation also in the previous line, `σ(::AbstractVector{Float64}, ::ZeroSet)`?